ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/JiraIntegrationForm.tsx

```diff
@@ -388,7 +404,7 @@ function JiraIntegrationForm({
                                                                     }
                                                                     onChange={onChange}
                                                                     onBlur={handleBlur}
-                                                                    isDisabled={!isEditable}
+                                                                    isDisabled={values.notifier.jira.disablePriority}
```

Comment:
This needs to be `isDisabled={!isEditable || values.notifier.jira.disablePriority}` correct?